root/layout/components/sidebar/WorkSidebar.js

```diff
@@ -78,6 +80,15 @@ const WorkSidebar = ({$c, work}: Props) => {
               </SidebarProperty>
             ) : null}
 
+            {length ? (
+              <SidebarProperty
+                className="work-length"
+                label={addColonText(l('Median Length'))}
+              >
+                {formatTrackLength(length)}
+              </SidebarProperty>
```

Comment:
Since there are a lot of wrong durations, I suggest to either put a disclaimer here that this is experimental, or show in work report instead of work page.